packages/nc-gui/components/smartsheet/details/Api.vue

```diff
@@ -95,7 +101,7 @@ const code = computed(() => {
 const api = new Api({
   baseURL: "${(appInfo.value && appInfo.value.ncSiteUrl) || '/'}",
   headers: {
-    "xc-auth": ${JSON.stringify(token.value as string)}
+    "xc-token": "<CREATE YOUR API TOKEN FROM : ${location.origin + location.pathname}#/account/tokens>"
```

Comment:
here `>` at the end may confuse and user copy the URL with `>` and it won't work. May be it's better to add a space or provide a link in the page itself.